src/main/java/com/ldtteam/structurize/blocks/PlaceholderBlock.java

```diff
@@ -45,7 +44,7 @@ public ActionResultType onBlockActivated(
     {
         if (worldIn.isRemote)
         {
-            new WindowPlaceholderblock(pos).open();
+            // new WindowPlaceholderblock(pos).open();
         }
```

Comment:
Why?

Reply:
its broken and placeholder functionality isnt used anymore

Reply:
Then just remove this if completely